doc/source/tune-usage.rst

```diff
@@ -296,6 +296,43 @@ of a trial, you can additionally set the checkpoint_at_end to True. An example i
         },
     })
 
+Recovering From Failures (Experimental)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tune automatically persists the progress of your experiments, so if an experiment crashes or is otherwise cancelled, it can be resumed by passing `resume=True` in `run_experiments()`. If resume is not enabled, a new experiment will be created instead. You can always force a new experiment to be created by changing the experiment name.
```

Comment:
```suggestion
 Tune automatically persists the progress of your experiments, so if an experiment crashes or is otherwise cancelled, it can be resumed by passing `resume=True` in `run_experiments()`. If `resume=False`, a new experiment will be created instead. The default setting of `resume=None` will cause Tune to prompt you for whether you want to resume. You can always force a new experiment to be created by changing the experiment name.
```

Reply:
done (did it before I saw this comment)